docs/api/networkinterfaces/WifiInterface.md

```diff
@@ -56,11 +56,11 @@ Please note that settings should match the security mode from the access point.
 
 ### Wi-Fi example
 
-Here is an example of an HTTP client program. The program brings up an ESP8266 as the underlying network interface, and uses it to perform an HTTP transaction over a TCPSocket. There are multiple [Wi-Fi components](https://os.mbed.com/components/cat/wifi/) that implement the WiFiInterface class. The example below uses the [ESP8266Interface](https://github.com/armmbed/esp8266-driver) and [OdinWiFiInterface](https://github.com/u-blox/ublox-odin-w2-drivers-docs-mbed-5).
+Here is an example of a program that scans for the Wi-Fi access points in the vicinity. There are multiple [Wi-Fi components](https://os.mbed.com/components/cat/wifi/) that implement the WiFiInterface class. The example below uses the [ESP8266Interface](https://github.com/armmbed/esp8266-driver) and [OdinWiFiInterface](https://github.com/u-blox/ublox-odin-w2-drivers-docs-mbed-5).
```

Comment:
This program scans for Wi-Fi access points in the area.

Reply:
Done